app/assets/javascripts/components/profile_card.es6.jsx

```diff
@@ -1,12 +1,26 @@
 class ProfileCard extends React.Component {
 
+  static getpropTypes() {
+    return {
+      cardName: React.PropTypes.string,
+      cardBody: React.PropTypes.string,
+    }
+  }
+
+  static get defaultProps() {
+    return {
+      cardName: '',
+      cardBody: '',
+    }
+  }
+
   get styles() {
     return {
       container: {
         display: 'inline-block',
         position: 'relative',
-        width: 172,
-        padding: 24,
+        width: '33%',
+        height: '250px',
```

Comment:
Don't need the `px` when dealing with width and height constants. Instead of `250px` you can do `250`.